ReSharper.FSharp/src/FSharp.Psi.Services/src/Generate/GenerateSignatureProvider.fs

```diff
@@ -177,8 +177,8 @@ type FSharpGenerateSignatureBuilder() =
                         if isNotNull symbolUse then
                             let mfv = symbolUse.Symbol :?> FSharpMemberOrFunctionOrValue
 
-                            mfv.Attributes
-                            |> Seq.iter (fun a -> sb.Append($"{a.Format(symbolUse.DisplayContext)}{lineEnding}") |> ignore)
+                            binding.Attributes
+                            |> Seq.iter (fun a -> sb.Append($"[<{a.GetText()}>]{lineEnding}") |> ignore)
```

Comment:
Why do you prefer to do a fresh parsing instead of copying the nodes?
Does it handle attribute targets and indentation properly?

Reply:
Sorry, could you elaborate? I'm pretty much out of my depth here.
How should I copy the nodes to build up the string we need?

Reply:
It seems there should be no reason to build a string of the attributes in the first place, if you already have the desired nodes. Please see what this PR already does for `open` nodes.